texastribune/donations-app
# End-of-year campaign page

#### What's this PR do?
Updates `/campaign.html` with some end-of-year materials highlighting some of the Trib's investigative projects.

#### Why are we doing this? How does it help us?
Membership request.

#### Have you done the following membership-specific things (if necessary)?
* [x] Update campaign ID
* [x] If launching a campaign, redirect `index.html` to `campaign.html` (probably)
* [ ] If ending a campaign, redirect `campaign.html` to `index.html` (probably)
* [x] Update social meta tags
* [ ] After deploy, [bust](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/troubleshooting-cards) the Twitter card cache

#### How should this be manually tested?
+ `make`
+ `yarn run dev`
+ Go to `http://localhost:4567/campaign.html` and make sure it looks OK
+ Go to `/index.html` and confirm it redirects

#### Have automated tests been added?
No.

#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?
Not really.

#### What are the relevant tickets?
Off-sprint, but here's the marketing ticket:
https://3.basecamp.com/3098728/buckets/5485001/todos/817062439#__recording_818400294

#### How should this change be communicated to end users?
No communication needed.

#### Next steps?
NA.

#### Smells?
NA.

#### TODOs:
* [ ] Art approval

#### Has the relevant documentation/wiki been updated?
NA.

#### Technical debt note
Same.
